DAO.sol

```diff
@@ -855,6 +878,7 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
     function minQuorum(uint _value) internal constant returns (uint _minQuorum) {
         // minimum of 20% and maximum of 53.33%
+        uint combinedTotalSupply = DAO(privateCreation).totalSupply() + totalSupply - tokensReceived;
```

Comment:
can we rename `privateCreation` to `parentDAO` or something. As it is pretty confusing.